Could we update the ARM build instructions, please?

I'm on the latest Ubuntu and first tried the "whonix repo method" to get the binary, even though I see the ARM parameter somewhere, it only managed to get me x86 binaries itself.

I also followed the standard Ubuntu recipe with make release, and it broke with the -maes unknown error, which has already been discussed in the RandomX context a long time ago, if not in this repo as well. As -maes simply should be removed for ARM, I looked into the Makefile only to find a release-linux-armv8 target. So I guessed that should do the job, except that it does not behave like make release, has some hardcoded dependency to ../.. for whatever reason. I did try the standard cmake recipe but it breaks almost immediately. 

Finally, I created bin/arm , copied the Makefile there, and now the stars seem to have aligned and the project gets built correctly in monero-gui/bin/arm/bin . Perhaps someone more competent than me knows what to fix here or there to make the build more streamlined.

I can pick it up here since I'm now at that point.  monero_gui/Makefile has,
>release:
 >       mkdir -p $(builddir)/release && cd $(builddir)/release && cmake -D DEV_MODE=$(or ${DEV_MODE},OFF) -DMANUAL_SUBMODULES=${MANUAL_SUBMODULES} -D ARCH="x86-64" -D CMAKE_BUILD_TYPE=Release $(topdir) && $(MAKE)
>
>release-linux-armv8:
>       mkdir -p $(builddir)/release
>        cd $(builddir)/release
>        cmake -D DEV_MODE=$(or ${DEV_MODE},OFF) -D ARCH="armv8-a" -D BUILD_64=ON -D CMAKE_BUILD_TYPE=Release -D BUILD_TAG="linux-armv8" $(topdir) && $(MAKE)

So it's using `$(topdir)` in either case, and that's set to `../..` earlier in the Makefile.
>topdir := ../..

Then the build fails with,
>cmake -D DEV_MODE=OFF -D ARCH="armv8-a" -D BUILD_64=ON -D CMAKE_BUILD_TYPE=Release -D BUILD_TAG="linux-armv8" ../.. && make
>CMake Error: The source directory "/home/rock/build-area/monero-gui" does not appear to contain CMakeLists.txt.
>Specify --help for usage, or press the help button on the CMake GUI.
>make: *** [Makefile:53: release-linux-armv8] Error 1
>==> ERROR: A failure occurred in build().
>    Aborting...

Changing it to `topdir:= .` allows it to start building.

--- a/Makefile
+++ b/Makefile
@@ -48,9 +48,7 @@ release:
 	mkdir -p $(builddir)/release && cd $(builddir)/release && cmake -D DEV_MODE=$(or ${DEV_MODE},OFF) -DMANUAL_SUBMODULES=${MANUAL_SUBMODULES} -D ARCH="x86-64" -D CMAKE_BUILD_TYPE=Release $(topdir) && $(MAKE)
 
 release-linux-armv8:
-	mkdir -p $(builddir)/release
-	cd $(builddir)/release
-	cmake -D DEV_MODE=$(or ${DEV_MODE},OFF) -D ARCH="armv8-a" -D BUILD_64=ON -D CMAKE_BUILD_TYPE=Release -D BUILD_TAG="linux-armv8" $(topdir) && $(MAKE)
+	mkdir -p $(builddir)/release && cd $(builddir)/release && cmake -D DEV_MODE=$(or ${DEV_MODE},OFF) -D ARCH="armv8-a" -D BUILD_64=ON -D CMAKE_BUILD_TYPE=Release -D BUILD_TAG="linux-armv8" $(topdir) && $(MAKE)
 
 release-static:
 	mkdir -p $(builddir)/release && cd $(builddir)/release && cmake -D STATIC=ON -D DEV_MODE=$(or ${DEV_MODE},OFF) -DMANUAL_SUBMODULES=${MANUAL_SUBMODULES} -D ARCH="x86-64" -D BUILD_64=ON -D CMAKE_BUILD_TYPE=Release $(topdir) && $(MAKE)

